lib/migration-steps/index.js

```diff
@@ -92,6 +92,20 @@ class ContentType extends DispatchProxy {
 
     this.dispatch(actionCreators.field.delete(this.id, this.instanceId, id, fieldInstanceId, callsite));
   }
+
+  changeFieldId (oldId, newId) {
+    const callsite = getFirstExternalCaller();
+    const fieldInstanceId = this.fieldInstanceIds.getNew(oldId);
+    this.dispatch(actionCreators.field.update(
```

Comment:
Shouldn't we emit a new action type 'field rename'?
That way things would be very explicit and there's no special treatment based on a certain property (`newId`) being updated.

The actual payload change would then be done in a way similar to how we transform the field deletion into two updates.